### ISSUE TITLE
Docs: Moving GitHub/Bitbucket content to disconnected system

### ISSUE THREAD

**Issue by christophermay07** [cjnovak98/ocp4-disconnected-config#25]
From Install.md: "7. Clone the Repository:

      ```bash
      git clone https://github.com/cjnovak98/ocp4-disconnected-config
      ```
"

Redline text: "These steps - they seem like they're referring to the internet connected/staging box, which the syscon will never be. The above steps all seem to apply to the syscon. There's no context change/designation change for these steps"

Please update the docs to clarify how the disconnected config will make it to the target system.

**Comment by mriensch** [comment]
Makes sense. will push update with it. (not in this context, but i got the jist of what issue is)
